CHANGELOG.md

```diff
@@ -355,7 +355,7 @@ Versioning](https://semver.org/spec/v2.0.0.html).
 - Supports the TLS protocol.
 - Added migration functionality in `0.12.0`. This feature adds values for new fields
   (`orig_filenames`, `orig_mime_types`, `resp_filenames`, `resp_mime_types`) to
-  `http raw event` in versions 0.12.0 and earlier.
+  `Http` GraphQL type in versions 0.12.0 and earlier.
```

Comment:
I apologize for any confusion. The migration part is not related to GraphQL, so please change the existing "http raw event" to "Http".

Reply:
Modification completed. Thank you for the clarification, and apologies for the confusion earlier.